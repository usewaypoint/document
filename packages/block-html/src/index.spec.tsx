import React from 'react';

import { render } from '@testing-library/react';

import { Html } from '.';

describe('block-html', () => {
  it('renders with default values', () => {
    expect(render(<Html />).asFragment()).toMatchSnapshot();
  });
});
