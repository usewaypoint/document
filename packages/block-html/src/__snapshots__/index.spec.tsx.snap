// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`block-html renders with default values 1`] = `
<DocumentFragment>
  <div />
</DocumentFragment>
`;
